app/src/main/java/de/tum/in/tumcampusapp/service/DownloadService.kt

```diff
@@ -111,7 +115,9 @@ class DownloadService : JobIntentService() {
         val newsSuccess = downloadNews(force)
         val eventsSuccess = downloadEvents()
         val topNewsSuccess = downloadTopNews()
-        return updateNoteSuccess && cafeSuccess && kinoSuccess && newsSuccess && topNewsSuccess && eventsSuccess
+        val gradesSuccess = downloadGrades()
+        return updateNoteSuccess && cafeSuccess && kinoSuccess
+                && newsSuccess && topNewsSuccess && eventsSuccess && gradesSuccess
```

Comment:
uggh, this class is getting uglier by the number of times we are editing it. We should really write up a todo, to finally fix it.

Reply:
I tried to break it up a little bit in  #1188.
Really doesn't make a lot of sense having it all in one big service :man_shrugging: